### PR TITLE
Viewing 3D-printing requests

### PR DESCRIPTION
**Pull request by ThaiQ**
Added an option in the profile page that allow the user to look at their 3D-printing-request.

### CODE REVIEW

**Comment by ThaiQ** [comment]
Button
image

Modal:
image

Also tried to fix the responsive issue of the page
image

**Comment by ThaiQ** [comment]
Made 3 changes base on window size:
1. Full screen
image

2. Medium
image

3. Small (compatible with phones)
image

File: src/Pages/Profile/MemberView/profile-modifier.css [hunk 1]
```diff
@@ -45,5 +46,54 @@
...
+  margin-left: 10%;
+}
+
+@media screen and (max-width: 1200px) {
+  #profileLogo {
+    height: 15%;
+    left: 60%;
+  }
+
+  #innerTextTop{
```
Comment (evanugarte) [inline src/Pages/Profile/MemberView/profile-modifier.css]:
```suggestion
  #inner-text-top{
```

File: src/Pages/Profile/MemberView/profile-modifier.css [hunk 2]
```diff
@@ -45,5 +46,54 @@
...
+    font-size: 20px;
+  }
+
+  #profileBox {
```
Comment (evanugarte) [inline src/Pages/Profile/MemberView/profile-modifier.css]: Please follow css naming conventions from now on, no camelcase
```suggestion
  #profile-box {
```

File: src/Pages/Profile/MemberView/profile-modifier.css [hunk 3]
```diff
@@ -45,5 +46,54 @@
...
+    left: 30%;
+  }
+}
+
+@media screen and (max-width: 1000px) {
+  #profileLogo {
+    height: 0%;
+  }
+
+  #innerTextTop{
```
Comment (evanugarte) [inline src/Pages/Profile/MemberView/profile-modifier.css]:
```suggestion
  #inner-text-top{
```

File: src/Pages/Profile/MemberView/profile-modifier.css [hunk 4]
```diff
@@ -45,5 +46,54 @@
...
+    font-size: 20px;
+  }
+
+  #profileBox {
```
Comment (evanugarte) [inline src/Pages/Profile/MemberView/profile-modifier.css]:
```suggestion
  #profile-box {
```

File: src/Pages/Profile/MemberView/profile-modifier.css [hunk 5]
```diff
@@ -45,5 +46,54 @@
...
+    position: relative;
+    width: 100%;
+    top: 0%;
+    left: 0%;
+  }
+
+  #clip {
```
Comment (evanugarte) [inline src/Pages/Profile/MemberView/profile-modifier.css]: why is #clip and #clip2 the same with their styles? consolidate them into one.

**Pull request merged**